### PR TITLE
The Tip Not For You

### PR DESCRIPTION
**Pull request by Shadow-Quill**
Adds a silly tip:
`Some people are unable to read text on a game where half of it is based on text.`

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
I don't get it

**Comment by variableundefined** [comment]
...Not sure what the point is

**Comment by Lunaimon** [comment]
Um... line 21.

File: strings/sillytips.txt [hunk 1]
```diff
@@ -29,4 +29,5 @@ Absolutely nothing unusual will happen this shift. Promise.
 Send help. Trapped in tip writing factory.
 There is no such thing as a cow level.
 HONK.
-Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
\ No newline at end of file
+Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
+Some people are unable to read text on a game where half of it is based on text.
```
Comment (Ty-Omaha) [inline strings/sillytips.txt]: This is already in the code on line 21 as @Lunaimon pointed out

Comment (Couls) [reply strings/sillytips.txt]: I can't help but feel like this is ironic

**Comment by Shadow-Quill** [comment]
*scream

**Pull request closed without merging**